### PR TITLE
OpenTracing shim: baggage of active span was not used

### PR DESCRIPTION
**Pull request by zeitlinger**
OpenTracing shim: baggage of active span was not used

### CODE REVIEW

File: opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java [hunk 1]
```diff
@@ -135,6 +135,9 @@ void activateSpan_withBaggage() {
 
       assertThat(io.opentelemetry.api.trace.Span.current()).isSameAs(actualSpan);
       assertThat(io.opentelemetry.api.baggage.Baggage.current()).isSameAs(actualBaggage);
+
+      Span child = tracerShim.buildSpan("child").start();
+      assertThat(child.getBaggageItem("foo")).isNotNull();
```
Comment (jkwatson) [inline opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java:140]: it should be equal to `bar` shouldn't it?

Comment (zeitlinger) [reply opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java]: right - fixed

**Review by carlosalberto — APPROVED** [review]

**Comment by carlosalberto** [comment]
This is a correct, albeit incomplete Baggage support - good reminder that I need to wrap up a related PR I had somewhere ;) Thanks @zeitlinger and will be pushing such remaining changes next week ;)

**Comment by carlosalberto** [comment]
@open-telemetry/java-approvers

**Comment by jkwatson** [comment]
> Can we merge this one? I will create a follow up PR in a bit to add test cases and other checks related to this.

Does this need to go into this week's release, or should we want until we have all the changes in place before releasing?

**Comment by carlosalberto** [comment]
@jkwatson I'd definitely like to get this in ASAP to get some validation. The rest of the items related to Baggage are additional checks and other items (and I may have a PR later today if this gets merged soon, etc).

Any opinion @zeitlinger ?

**Comment by zeitlinger** [comment]
> @jkwatson I'd definitely like to get this in ASAP to get some validation. The rest of the items related to Baggage are additional checks and other items (and I may have a PR later today if this gets merged soon, etc).
> 
> Any opinion @zeitlinger ?

I'm already using this feature on a fork, so there's no time pressure on my side - but I'd welcome the integration, as it makes maintenance easier for me.

**Pull request merged**